### ISSUE TITLE
Upgrade Hibernate to 6.4

### ISSUE THREAD

**Issue by borsch** [starnowski/posjsonhelper#92]
Currently hibernate 6 support is based on version 6.1. Between 6.1 & 6.4 there was API change due to which NoSuchMethodError occurs

Both `AbstractJsonBExtractPath` & `AbstractJsonbArrayStringsExistPredicate` relies on method [HibernateCriteriaBuilder.value 6.1](https://docs.jboss.org/hibernate/orm/6.1/javadocs/org/hibernate/query/criteria/HibernateCriteriaBuilder.html#value(T)) which has result type as `SqmExpression<T>` but in Hibernate 6.4 this method has result type as `JpaExpression<T>` [HibernateCriteriaBuilder.value 6.4](https://docs.jboss.org/hibernate/orm/6.4/javadocs/org/hibernate/query/criteria/HibernateCriteriaBuilder.html#value(T)) due to which following error occurs

```
Caused by: java.lang.NoSuchMethodError: 'org.hibernate.query.sqm.tree.expression.SqmExpression org.hibernate.query.sqm.NodeBuilder.value(java.lang.Object)'
	at com.github.starnowski.posjsonhelper.hibernate6.AbstractJsonBExtractPath.lambda$mapPathParameters$0(AbstractJsonBExtractPath.java:82)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.AbstractList$RandomAccessSpliterator.forEachRemaining(AbstractList.java:720)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682)
	at com.github.starnowski.posjsonhelper.hibernate6.AbstractJsonBExtractPath.mapPathParameters(AbstractJsonBExtractPath.java:82)
	at com.github.starnowski.posjsonhelper.hibernate6.AbstractJsonBExtractPath.<init>(AbstractJsonBExtractPath.java:63)
	at com.github.starnowski.posjsonhelper.hibernate6.JsonBExtractPathText.<init>(JsonBExtractPathText.java:39)
```

**Comment by starnowski** [comment]
Hello @borsch,

Thank you for reporting this issue. The fix was added in release 0.3.0. Please try to update the project version 0.3.0 and check if works.

**Comment by borsch** [comment]
Hi @starnowski 
Seems to be working fine. Thanks for quick resolution!
